libs/utils/executor.py

```diff
@@ -477,21 +477,21 @@ def _wload_init(self, tc, wl_idx):
         wload = self._wload_conf(wl_idx, wlspec)
 
         # Keep track of platform configuration
-        self.te.test_dir = '{}/{}:{}:{}'\
+        test_dir = '{}/{}:{}:{}'\
             .format(self.te.res_dir, wload.wtype, tc_idx, wl_idx)
-        os.system('mkdir -p ' + self.te.test_dir)
-        self.te.platform_dump(self.te.test_dir)
+        os.system('mkdir -p ' + test_dir)
```

Comment:
surely you mean 92440b8875612b0f7eb6fd7a82d3b9939cbf28c8 ("Use os.makedirs instead of os.system('mkdir ...)") (thanks github :disappointed: )